Sources/CLibMongoC/include/CLibMongoC_bson-version.h

```diff
@@ -31,30 +31,30 @@
  *
  * BSON minor version component (e.g. 2 if %BSON_VERSION is 1.2.3)
  */
-#define BSON_MINOR_VERSION (16)
+#define BSON_MINOR_VERSION (17)
```

Comment:
I opened [SWIFT-1122](https://jira.mongodb.org/browse/SWIFT-1122) about keeping these updated in line with the release script